streaming/multimodal/convert/webvid/crawl.py

```diff
@@ -43,23 +43,35 @@ def parse_args() -> Namespace:
         Command-line arguments.
     """
     args = ArgumentParser()
-    args.add_argument('--in',
-                      type=str,
-                      required=True,
-                      help='Dataset CSV file from https://m-bain.github.io/webvid-dataset/')
-    args.add_argument('--out',
-                      type=str,
-                      required=True,
-                      help='Path to MDS dataset that we will create')
-    args.add_argument('--num_procs',
-                      type=int,
-                      default=64,
-                      help='Number of processes to use for downloading videos')
+    args.add_argument(
+        '--in_root',
+        type=str,
+        required=True,
+        help='Dataset CSV file from https://m-bain.github.io/webvid-dataset/',
+    )
+    args.add_argument(
+        '--local',
```

Comment:
Input is prefixed with `in`, so outputs need to be prefixed as `out_local`, `out_remote`

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/171/commits/5b5c10450637d4ccdbc8b5c2403a7d18f89b169b.